daq/runner.py

```diff
@@ -523,10 +525,12 @@ def _handle_device_learn(self, target_mac, vid):
         else:
             self._target_set_trigger(device)
 
-    def _remote_trigger(self, device, device_vlan, assigned_vlan):
+    def _remote_trigger(self, device, device_vlan, assigned_vlan, device_port):
         assert device_vlan, 'expected device_vlan'
         device.port.flapping_start = 0
         device.port.active = True
+        if device_port:
```

Comment:
this doesn't seem right.  The device is on a target_port no matter what the of situation

Reply:
Changed.